jupyter_server/extension/application.py

```diff
@@ -140,6 +141,11 @@ def static_url_prefix(self):
         help=_("The default URL to redirect to from `/`")
     )
 
+    open_browser = Bool(
+        True,
+        help=_("Should the extension open a browser window?")
+    )
+
```

Comment:
Do we want an alias on this open_browser config so we can use `jupyter server --open-browser False`?

```
    aliases.update({
        'open-browser': 'ExtensionApp.open_browser',
    })
```

Reply:
Hm, I think the `--no-browser` flag is probably enough.

The main `ServerApp` doesn't have an `open-browser` alias, so I think we should keep parity with that.